services/static-badge/static-badge.service.js

```diff
@@ -11,7 +11,7 @@ module.exports = class StaticBadge extends BaseStaticService {
   static get route() {
     return {
       base: '',
-      format: '(?::|badge/)((?:[^-]|--)*?)-?((?:[^-]|--)*)-((?:[^-]|--)+)',
+      format: '(?::|badge/)((?:[^-]|--)*?)-?((?:[^-]|--)*)-((?:[^-.]|--)+)',
```

Comment:
> It would be nice to replace this with a route with a more reasonable pattern, and maybe a second service for the :: form. Or, we could consider redirecting to /badge as the canonical format.

I find that pattern pretty tough on the eyes, so all on board with anything that helps simplify that